Sources/App/Common/Helpers/Constants/Constants.swift

```diff
@@ -19,5 +20,14 @@ struct Constants {
   
   struct Path {
     static let userPhotos = "user_photos"
+    static let creatorsPhotos = "creators"
```

Comment:
Maybe use `creator_photos` here or `users` in `userPhotos` for consistency?

Reply:
ok, changed to `creator_photos`